airgun/widgets.py

```diff
@@ -691,6 +692,7 @@ def select_loc(self, loc_name):
         )
         self.browser.plugin.ensure_page_safe()
 
+    @property
```

Comment:
should not you change https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/location.py#L87 too?